src/main/java/com/kodilla/ecommercee/domain/group/Group.java

```diff
@@ -0,0 +1,61 @@
+package com.kodilla.ecommercee.domain.group;
+
+import com.kodilla.ecommercee.domain.product.Product;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "group")
```

Comment:
Nazwa "group" jest zastrzeżona jako słowo kluczowe w bazie wiec to niezadziała

Reply:
ok, zmienię to.